pkg/controllers/cloud_config_sync_controller_test.go

```diff
@@ -81,43 +81,57 @@ func makeInfraStatus(platform configv1.PlatformType) configv1.InfrastructureStat
 	}
 }
 
-func makeInfraCloudConfig() *corev1.ConfigMap {
+func makeInfraCloudConfig(platform configv1.PlatformType) *corev1.ConfigMap {
+	defaultConfig := `[Global]
```

Comment:
nit: Is this changing the meaning of default to be an aws config, rather than azure?
I might be tempted to switch on platformtype here to make things explicit.